init.lua

```diff
@@ -26,11 +26,39 @@ local teleport_player_to_pocket = pocket_dimensions.teleport_player_to_pocket
 
 local pocket_size = pocket_dimensions.pocket_size
 
-local personal_pockets_chat_command = minetest.settings:get_bool("pocket_dimensions_personal_pockets_chat_command", false)
-local personal_pockets_key = minetest.settings:get_bool("pocket_dimensions_personal_pockets_key", false)
-local personal_pockets_key_uses = tonumber(minetest.settings:get("pocket_dimensions_personal_pockets_key_uses")) or 0
-local personal_pockets_spawn = minetest.settings:get_bool("pocket_dimensions_personal_pockets_spawn", false)
-local personal_pockets_respawn = minetest.settings:get_bool("pocket_dimensions_personal_pockets_respawn", false) and not minetest.settings:get_bool("engine_spawn")
+-- Attempt to load values, apply no default
+local personal_pockets_chat_command = minetest.settings:get_bool("pocket_dimensions_personal_pockets_chat_command")
+local personal_pockets_key = minetest.settings:get_bool("pocket_dimensions_personal_pockets_key")
+local personal_pockets_key_uses = tonumber(minetest.settings:get("pocket_dimensions_personal_pockets_key_uses"))
+local personal_pockets_spawn = minetest.settings:get_bool("pocket_dimensions_personal_pockets_spawn")
+local personal_pockets_respawn = minetest.settings:get_bool("pocket_dimensions_personal_pockets_respawn") and not minetest.settings:get_bool("engine_spawn")
+
```

Comment:
This changes it to where we attempt to load the values... if they fail then it will be equal to `nil` or `none`.